pages/common/assets/sass/pdf/index.scss

```diff
@@ -122,27 +112,10 @@ dl dd {
   text-indent: 5px;
 }
 
-.break {
-  page-break-after: always;
-}
```

Comment:
This one probably wants to stay.

Reply:
YTHO?

Reply:
if we don't want to throwing breaks around then surely this is doing exactly that?

Reply:
Because if there's an element with a classname of `.break` it implies that the page break is deliberate.

If it shouldn't have a page break then don't put the `.break` element in.